package.json

```diff
@@ -61,12 +62,13 @@
     "@types/lodash.union": "^4.6.6",
     "@types/sarif": "^2.1.3",
     "@types/uuid": "^8.3.0",
-    "axios": "^0.21.1",
     "ignore": "^5.1.8",
+    "ipaddr.js": "^2.0.1",
```

Comment:
You can remove `ipaddr.js` as it's not needed for getIpFamily() function any more.

Reply:
done